modules/auxiliary/dos/scada/siemens_siprotec4.rb

```diff
@@ -0,0 +1,36 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
```

Comment:
Also having a one new line between commented out section and class definition would be nice too.

Reply:
Done